test/test_unary_ufuncs.py

```diff
@@ -377,6 +378,36 @@ def test_batch_vs_slicing(self, device, dtype, op):
 
         self.assertEqual(actual, expected)
 
+    @dtypes(*(torch.testing.get_all_int_dtypes() + torch.testing.get_all_fp_dtypes(include_bfloat16=False)))
+    def test_nan_to_num(self, device, dtype):
+        for contiguous in [False, True]:
+            for x in generate_numeric_tensors(dtype=dtype, device=device):
+                if not contiguous:
+                    x = x.T
+                self.compare_with_numpy(torch.nan_to_num, np.nan_to_num, x)
```

Comment:
Will this and the next comparison be covered by the existing unary ufunc tests?

That is, I'm wondering if only the tests where the extra args are set are needed in this test?

Reply:
I am not sure about method variant but `torch.nan_to_num` is verified. You know where the

Reply:
Variants are tested for consistency here: https://github.com/pytorch/pytorch/blob/7cde662f08b226fb8d7f8e99dbd6f40dbe2dca16/test/test_unary_ufuncs.py#L200

Which is not the most thorough test, but we didn't want to run the tests again for method variants because it would greatly increase test time (and cost).

Reply:
Ah. Will remove those two then. Thanks!